profiler/src/ProfilerEngine/Datadog.Profiler.Native/StackSamplerLoop.cpp

```diff
@@ -25,12 +25,16 @@
 #include "SymbolsResolver.h"
 #include "ThreadsCpuManager.h"
 #include "IWallTimeCollector.h"
+#include "ICollector.h"
 #include "WallTimeSampleRaw.h"
+#include "RawCpuSample.h"
+#include "SystemTime.h"
 
 #include "shared/src/native-src/string.h"
 
 // Configuration constants:
 using namespace std::chrono_literals;
+uint64_t SamplingPeriodMs = 9; // TODO: keep this one in synch with SamplingPeriod
```

Comment:
you can remove this and at l.638 use `SamplingPeriod.count()`

Reply:
✅